konf/site.yaml

```diff
@@ -5,4 +5,4 @@ readinessPath: "/"
 demo:
   env:
     - name: MAAS_URL
-      value: "http://polong.internal:5240/"
+      value: "http://10.131.211.203:5240/"
```

Comment:
Should we be exposing IPs like this in a public repo? I feel like we should wait until we have an A record

Reply:
I created a [followup task](https://warthogs.atlassian.net/browse/MAASENG-2218) for this, but I think we can wait. We're not in a hurry.